### DISCUSSION TITLE
Dúvida sobre front

### DISCUSSION THREAD

**Discussion by sidneydasilva537** [frontendbr/forum#2026] [Discussão]
Olá, tenho um site que fiz recentemente no loja integrada, mas não posso acessar o backend para incluir gostaria de saber se posso incluir uma coluna para o cliente digitar a regiao no formulario de contato referente a certo item.

por exemplo:
https://www.spgeo.com.br/GeoMax-Zenith16-GNSS-RTK-Base-e-Rover

esse é um produto nosso, queria incluir no botão de consulte o preço um campo para ele colocar o endereço e um para a mensagem, tem como fazer isso só com html, css e javascript? pensei em fazer com o iframe mas eu não conseguiria saber o produto, o cliente teria que digitar.

**Comment by willycamargo** [comment]
Cara, não acho que o problema esta especificado de forma objetiva suficiente para ter uma resposta clara, e talvez até por isso ninguém tenha respondido ainda.

Você precisa adicionar mais 2 campos no formulários que aparece ao clicar no botão?

Algo assim?
![image](https://user-images.githubusercontent.com/5041887/108593830-e583cf80-7354-11eb-98e2-80dbe2722603.png)

Se for o caso, você precisará adicionar os campos no front e provavelmente alterar o back para suportar esses novos campos na chamada. Porém eu não conheço nada de Loja Integrada e não sei se ele serializa a chamada toda e salva todos os campos, porque dai no caso talvez não precisasse alterar o tratamento da chamada.

Acho que se você conseguir detalhar melhor o que você tem funcionando hoje e o que você precisa que aconteça, poderemos te ajudar melhor.

**Reply by sidneydasilva537** [reply]
Sim, é exatamente assim que eu gostaria, mas eles só permitem editar o front end da loja!  Tem algum modo de incorporar esse front ao site?

**Reply by willycamargo** [reply]
Desculpa a demora na resposta.

Acredito que seja possível de alguma forma sim @sidneydasilva537 , mas eu não faço a mínima ideia pois nunca trabalhei com loja integrada 😞 

Acho que talvez vale entrar em contato direto com suporte para te direcionarem para alguma documentação específica sobre isso.
